lib/frr_zmq.c

```diff
@@ -153,14 +153,13 @@ static int frrzmq_read_msg(struct thread *t)
 
 	thread_add_read(t->master, frrzmq_read_msg, cbp,
 			cb->fd, &cb->read.thread);
-	return 0;
+	return;
```

Comment:
missed, cleanup.

Reply:
I cannot clean this one up as that it would issue a flog_err and call back the cb_error function.